packages/core/src/attestation/Attestation.ts

```diff
@@ -240,6 +289,18 @@ export default class Attestation implements IAttestation {
    *
    * @param maxDepth - The number of levels to walk up the delegation hierarchy until the delegation node is found.
    * @returns A promise containing the unsigned SubmittableExtrinsic (submittable transaction).
+   * @example ```javascript
+   * attestation.remove(3).then((removalExtrinsic) => {
+   *   // The attestation removal tx was created, and it can now be signed by the attestation owner.
+   *   const attestationOwnerDid = did;
```

Comment:
C: This does not seem to be needed

Reply:
https://github.com/KILTprotocol/sdk-js/pull/434/commits/974293be648c3fafc24709faf09638530614cb5c